web/src/search/input/SearchPage.tsx

```diff
@@ -213,14 +213,14 @@ export const SearchPage: React.FunctionComponent<Props> = props => {
                                         Diff/commit search keywords:
                                     </dt>
                                     <dd className="text-monospace">type:diff or type:commit</dd>
-                                    <dd className="text-monospace">after:”2 weeks ago”</dd>
+                                    <dd className="text-monospace">after:2 weeks ago</dd>
```

Comment:
I think they were required, but the problem was that the quotes are stylized 🤔 
```suggestion
                                    <dd className="text-monospace">after:"2 weeks ago"</dd>
```

Reply:
At least I'm getting nothing for that one:
https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+after:180+minutes+ago+type:commit&patternType=literal

but that one works:
https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+after:%22180+minutes+ago%22+type:commit&patternType=literal

Reply:
Ah, I misunderstood. Thank you for clarifying - fixing!